### ISSUE TITLE
Hello. Can I use this project to build both Android and Windows App? 

### ISSUE THREAD

**Issue by RedStartGong** [rjuszczyk/ComposeVideoPlayer#1]

**Comment by LucasAlfare** [comment]
I that this project has as dependency some specific jetbrains desktop artifacts, I think. For this reason you can't use the same project directly from Android.

If you need, you can extend this project to multiplatform approach, then the code here wil be used by desktop and you can rewrite this same code without desktop specifics to match the Android platform.

**Comment by rjuszczyk** [comment]
Hey,

You are right, plus it uses the jvm version of vlc. I will update the repo to include multiplatform approach as I have it done in my pet project. Although for Android I will use androids media player as I find it better somehow.

Will try to push something this evening.

**Comment by rjuszczyk** [comment]
Hey, it was a bit more work than I expected and finding a free evening was a bit more difficult too, but finally I have updated it.

I have changed the API significantly, please have a look and let me know what do you think.

**Comment by LucasAlfare** [comment]
Thanks for the updates, however I can't see the actual implementations over the folders. Did you missed then on commiting?

I'm not with my computer now, then all I can do is read the code, for now. Let me know if is all ok with the repository.

**Comment by rjuszczyk** [comment]
Yes, I missed it because I was copying a diff to commit it from another computer where I was logged with this github account. Since the new files were not yet tracked they didnt get included in the diff and I missed that. 

I will update it as soon as I am on the other computer. Sorry for that!

**Comment by radodado** [comment]
should be resolved now

**Comment by LucasAlfare** [comment]
Thanks for the update! Now I think that should be fine use same API in desktop and Android.

Next days I'll create some tests in my application and see how this behave now.
